### ISSUE TITLE
At OpenShiftOperation.java:47 will close the context.getClient()

### ISSUE THREAD

**Issue by walnut-tom** [fabric8io/kubernetes-client#1762]
#### The OpenShiftClient extends AutoCloseable, so it's will be close
```
      try (OpenShiftClient oc = new DefaultOpenShiftClient(context.getClient(), config)) {
        if (config.isOpenShiftAPIGroups(oc)) {
          String apiGroupUrl = URLUtils.join(config.getMasterUrl(), "apis", context.getApiGroupName(), oapiVersion);
          String apiGroupVersion = URLUtils.join(context.getApiGroupName(), oapiVersion);
          return context.withConfig(new OpenShiftConfigBuilder(config).withOpenShiftUrl(apiGroupUrl).build()).withApiGroupName(context.getApiGroupName()).withApiGroupVersion(apiGroupVersion);
        } else {
          String apiGroupUrl = URLUtils.join(config.getMasterUrl(), "oapi", oapiVersion);
          return context.withConfig(new OpenShiftConfigBuilder(config).withOpenShiftUrl(apiGroupUrl).build()).withApiGroupName(context.getApiGroupName()).withApiGroupVersion(oapiVersion);
        }
      }
```

**Comment by rohanKanojia** [comment]
There is a PR #1761 to fix it.

**Comment by rohanKanojia** [comment]
@walnut-tom : We've fixed this and released 4.5.2, please try it out and share feedback. Closing this issue now, feel free to re-open if you still face this.
